packages/schema/cacheGeoJSONSchema.js

```diff
@@ -0,0 +1,29 @@
+const fs = require('fs');
+const https = require('https');
+const path = require('path');
+
+const schemaUrl = 'https://geojson.org/schema/GeoJSON.json';
+const cacheDir = path.resolve(__dirname, '.cache/schemas');
```

Comment:
Should we add an entry for `.gitignore` for this?

Reply:
We don't need to, since folder name is `.cache` and it's already there

Reply:
ok!